lib/rails2_ruby2/action_pack.rb

```diff
@@ -0,0 +1,3 @@
+Dir[File.dirname(__FILE__) + '/action_pack/**/*.rb'].each do |file|
+  require file.sub(/#{Regexp.escape(File.extname(file))}$/, '')
+end
```

Comment:
newlines eol!

Reply:
👍 ending newlines added to files that were missing it in 54c483e22a9c7d1eb26dd835ea44d770a1e1f8f5